config/checker/com.intellij.astub

```diff
@@ -137,6 +138,10 @@ class NotificationAction {
   static NotificationAction createSimple(Supplier<String> dynamicText, @UI Runnable performAction);
 }
 
+class NotificationGroup {
+      Notification createNotification(@Tainted String title, @Tainted String content, NotificationType type, @Nullable NotificationListener listener);
```

Comment:
Nit: two spaces for indent

Reply:
two spaces between package name and class or two spaces between each class ? :>

Reply:
Spaces! not newlines

2 spaces as one level indent